### PR TITLE
docs: translate Legacy React APIs

### LINKED ISSUE #545: Legacy React APIs
Page: https://id.react.dev/reference/react/legacy

### PR DESCRIPTION
**Pull request by renomureza**
Closes #545

## Description

Translate the Legacy React APIs page.
Page URL:  https://id.react.dev/reference/react/legacy

### CODE REVIEW

File: src/content/reference/react/legacy.md [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
-* [`Component`](/reference/react/Component) lets you define a React component as a JavaScript class. [See alternatives.](/reference/react/Component#alternatives)
-* [`createElement`](/reference/react/createElement) lets you create a React element. Typically, you'll use JSX instead.
-* [`createRef`](/reference/react/createRef) creates a ref object which can contain arbitrary value. [See alternatives.](/reference/react/createRef#alternatives)
-* [`isValidElement`](/reference/react/isValidElement) checks whether a value is a React element. Typically used with [`cloneElement`.](/reference/react/cloneElement)
-* [`PureComponent`](/reference/react/PureComponent) is similar to [`Component`,](/reference/react/Component) but it skip re-renders with same props. [See alternatives.](/reference/react/PureComponent#alternatives)
+## API Lama {/*legacy-apis*/}
 
+* [`Children`](/reference/react/Children) memungkinakn Anda memanipulasi dan mengubah JSX yang diterima sebagai *prop* `children`. [Lihat alternatif.](/reference/react/Children#alternatives)
```
Comment (resir014) [inline src/content/reference/react/legacy.md:15]:
```suggestion
* [`Children`](/reference/react/Children) memungkinkan Anda memanipulasi dan mengubah JSX yang diterima sebagai *prop* `children`. [Lihat alternatif.](/reference/react/Children#alternatives)
```

typo

**Review by resir014 — CHANGES_REQUESTED** [review]
One minor change. @renomureza

File: src/content/reference/react/legacy.md [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
 
 ---
 
-## Deprecated APIs {/*deprecated-apis*/}
+## API yang tidak digunakan lagi {/*deprecated-apis*/}
 
 <Deprecated>
 
-These APIs will be removed in a future major version of React.
+API ini akan dihapus di versi utama React yang akan datang.
```
Comment (mazipan) [inline src/content/reference/react/legacy.md:29]: Plural.
Major -> Mayor
```suggestion
API-API ini akan dihapus di versi mayor React yang akan datang.
```

**Review by mazipan — APPROVED** [review]

**Comment by mazipan** [comment]
This is quite small, let's ship this @resir014, need your approval

**Comment by renomureza** [comment]
Thanks for the correction, mas @resir014 @mazipan.

**Review by resir014 — APPROVED** [review]

**Pull request merged**